pgcli/pgexecute.py

```diff
@@ -289,7 +289,9 @@ def execute_normal_sql(self, split_sql):
         cur.execute(split_sql)
 
         try:
-            title = self.conn.notices.pop()
+            title = ''
+            while len(self.conn.notices) > 0:
+                title = title + self.conn.notices.pop()
```

Comment:
I'm not sure why I originally decided to used pop() on the notices. But I think we have an opportunity to fix it. Let's make `title = '\n'.join(self.conn.notices)`.

I think that'll be faster and more idiomatic.